lib/tasks/ingest.rake

```diff
@@ -515,9 +553,9 @@ namespace :ingest do
         File.foreach(path) do |line|
           next if $. == 1
 
-          # This appears in one of the dumps: 
+          # This appears in one of the dumps:
           # "p55258827"|"151"|"EDWARDS, TESNA"|"27777016362867"|"zx080"|"TEDWARD2@SCHOOLS.NYC.GOV";"TEdward2@schools.nyc.gov"
```

Comment:
Let's definitely delete the user's email info, and clean the history.  Sorry this is still a bother, but hopefully we'll be finding fewer such history-cleaning triggers in the future.  Maybe let's do a global search for any other email address mentions.  We want to do the cleaning before the commit histories get interwoven into branch histories.

For this comment doc string, which is useful in explaining the following bit of code, let's replace the user's real email and name with a mock, like "JANEDOE@SCHOOLS.NYC.GOV".

Reply:
@nnat425 Darya suggested creating a new Git repo once this change is rolled in.  Since you have more PRs in this repo, I think it makes sense for you to decide on the best timing.